services-implemented-version/backend-for-frontend-python/server.py

```diff
@@ -35,4 +35,4 @@ def create_picture():
         return flask_response
 
 if __name__ == '__main__':
-    app.run(debug=True, port=10114)
+    app.run(port=10114)
```

Comment:
Should this be port 10115?

```suggestion
    app.run(port=10115)
```